src/app/downloads/downloads.component.html

```diff
@@ -27,8 +24,8 @@ <h3 class="col-12 mb-2 mt-4 card-subtitle text-muted">Select your platform to st
           <div class="col-xs-12 col-sm-2 col-lg-3">
             <a class="btn btn-default btn-block btn-primary btn-bg-dark mb-1" role="button" [href]="latestRelease.mac12" (click)="trackDownload('mac', latestRelease.mac12)">MacOS 12</a>
             <div class="alt-builds">
-              <a class="card-link text-center ml-2" [href]="latestRelease.mac11" (click)="trackDownload('linux', latestRelease.mac11)">MacOS 11</a>
-              <a class="card-link text-center ml-2" [href]="latestRelease.mac10" (click)="trackDownload('linux', latestRelease.mac10)">MacOS 10</a>
+			  <a class="card-link text-center ml-2" [href]="latestRelease.mac12_arm" (click)="trackDownload('mac', latestRelease.mac12_arm)">MacOS 12 ARM</a>
```

Comment:
I took off mac10, and added mac12_arm.  I think I updated all the necessary spots, but definitely worth a peek.